src/main/webapp/loginsuccess.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
This file likely won't be needed but good for a demo for the first week's work. In the future (this week's version), I think it'd be better if we redirected the user to the page where the posts are displayed. So for now this is good.